webapp-django/crashstats/settings/base.py

```diff
@@ -346,6 +347,12 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # Rate limit for when using the Web API for anonymous hits
 API_RATE_LIMIT = '10/m'
 
+# Rate limit when using the supersearch web interface
+RATELIMIT_SUPERSEARCH = '5/m'
```

Comment:
Let's change to `10/m` then. 

Also, being signed in it raises the limit to infinite which is something rather weird because anybody can bother to sign in. It doesn't give them any permissions.